### PR TITLE
Rasoa Kininike

### PR DESCRIPTION
**Pull request by Betsimisaraka**
Hey team

I have done this PR, please have a look at it and happy to change if need to be.

Thanks in advance
Anita

### CODE REVIEW

File: r/rasoa-kininike.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Rasoa Kininike
+
+![Rasoa Kininike](rasoa-kininike.JPG)
+
+This artist is born in 1978 and she was died in 2014 by bus accident. She is from Tongobory-Toliara.She is Lico Kininike's mother.
```
Comment (ndorfin) [inline r/rasoa-kininike.md]: `This artist was born in 1978 and died in 2014 by bus accident. She is from Tongobory-Toliara. She is Lico Kininike's mother.`

**Review by ndorfin — CHANGES_REQUESTED** [review]

**Comment by Betsimisaraka** [comment]
I have fixed the sentences, please have a look.

File: r/rasoa-kininike.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Rasoa Kininike
+
+![Rasoa Kininike](rasoa-kininike.JPG)
+
+This artist was born in 1978 and died in 2014 by bus accident. She is from Tongobory-Toliara.She is Lico Kininike's mother.
```
Comment (ndorfin) [inline r/rasoa-kininike.md]: Spaces after punctuation!

**Review by ndorfin — CHANGES_REQUESTED** [review]

**Comment by Betsimisaraka** [comment]
I have put space after punctuation.

File: r/rasoa-kininike.md (hunk 2 above)
Comment (Betsimisaraka) [reply r/rasoa-kininike.md]: I have put space after punctuation

**Review by ndorfin — APPROVED** [review]
Looks good.

**Pull request merged**